### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by hribeir0** [SafeExamBrowser/seb-win-refactoring#145] [Q&A]
Hi Daniel, 

I've had the issues presented here https://sourceforge.net/p/seb/discussion/844844/thread/11bf520f0e/?page=1 Even after installing manually all the dependencies. This happened at least in 3 different windows 10 computers. 

I was able to install version https://sebdev-let.ethz.ch/api/buildjobs/tgvq4q38nexl26m0/artifacts/SEB_3.2.0.324_x64_Setup.msi

Having in mind Moodle integration  we might use 3.2.0.338 or 3.2.0.324 correct?

**Comment by dbuechel** [comment]
To which issue are you referring exactly? In case you refer to the error message "Element not found" in the setup bundle which is caused by a failure to verify the WebView2 dependency: Either use the today released latest version 3.3.0 or then proceed as mentioned in #139.
